.NET/Microsoft.Recognizers.Text.Sequence/Extractors/BasePhoneNumberExtractor.cs

```diff
@@ -111,6 +111,17 @@ public override List<ExtractResult> Extract(string text)
                 if (er.Start != 0)
                 {
                     var ch = text[(int)(er.Start - 1)];
+                    var front = text.Substring(0, (int)er.Start);
+
+                    if (this.config.ForbiddenPrefixRegex != null &&
+                            this.config.ForbiddenPrefixRegex.IsMatch(front))
+                    {
+                        ers.Remove(er);
+                        i--;
+                        continue;
+                    }
+
+                    front = text.Substring(0, (int)(er.Start - 1));
```

Comment:
Nit: These small changes will make it be called more often than before, which slowly can lead to performance degradation. If not really necessary, I recommend not making such changes in unrelated PRs.

Reply:
And why "front" twice with one index difference? Please try to make the code more readable.